sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -256,7 +256,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[0] + '.txt',
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + '.txt',
```

Comment:
OK, I tried something: 71dd8bfbf94417ad55b2444e1dbd219db266f335.

It seems to work, but I don't know if that's the right way to do it ...